concepts/routing/README.md

```diff
@@ -0,0 +1,55 @@
+# Routes
+
+
+
+## What is a Route
+
+### Introduction
+A route is the functionality that implement the possibility to distinguish on URL from an another.
+Like all web frameworks, Trails route provide the mechanism that map route to controllers.
+
+## How to use it in Trails
+
+#### 1 - Configuration
+
+##### The common way
+
+```JavaScript
+// config/routes.js
+module.exports = [
+  {
+    method: [ 'GET' ],
+    path: '/example/test',
+    handler: 'ExampleController.test'
+  }
+]
+```
+
+###### Required values
+
+> method: can be ['POST', 'GET', 'PUT', 'DELETE']
```

Comment:
not only this ^^ OPTIONS, PATCH, HEAD are available too (don't remember if there others)